INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
 Qiita installation
 ==================
 
-Qiita is pip installable, but depends on some non-python packages that must be installed first.
+Qiita is pip installable, but depends on some non-python packages that must be installed first. We strongly recommend using virtual environments; the solution we recommend ot manage them is [miniconda](http://conda.pydata.org/miniconda.html), a lightweight version of the virtual environment, python distribution, and package manager anaconda.
+
+These instructions were tested successfully by [@HannesHolste](github.com/HannesHolste) on Mac OS X El Capitan 10.11.4 and a clean installation of Ubuntu 12.04 LTS (precise) using conda v4.0.7 and cloning from [qiita master branch commit #a9e4e03](https://github.com/biocore/qiita/commit/a9e4e03ecd781d3985abc03d15f2248143e565d7) on 5/27/2016.
+
+## Install miniconda
+
+Download the appropriate installer [here](http://conda.pydata.org/miniconda.html) corresponding to your operating system and execute it.
+
+Ensure that your `~/.bash_profile` (for default bash users) and/or `~/.zshrc` (for zshell users) contains the following line, which prepends a path leading to miniconda's binaries to the `$PATH` variable.
+
+```bash
+# added by Miniconda2 installer
+export PATH="/Users/hannes/miniconda2/bin:$PATH"
+```
+
+This essentially allows miniconda's binaries representing each of your virtual environments to take precedence over any custom, pre-existing installations of pip or python you might have installed through homebrew, for example – thus you do *not* have to uninstall them.
+
+Next, ensure conda is up-to-date.
+
+```bash
+conda update conda
+```
+
+Finally, restart your shell or reload your profile/rc file to ensure your `$PATH` variable was updated:
+
+````bash
+source ~/.bash_profile
+````
+
+## Setup miniconda environment for QIITA
+
+To ensure that QIITA can install its dependencies – some of which require older or specific distributions of pip packages – without interfering with any pre-existing, globally installed pip packages you might have, you will setup a virtual environment in conda that you must *activate* whenever you are working with or running QIITA.
```

Comment:
Agree with @antgonza 

This make sound Qiita bad, but that is a common thing on software packages, I would remove this sentence completely and argument that miniconda is recommended because Qiita and its plugins may have conflicting dependencies.